performancetest/src/main/java/org/corfudb/performancetest/LogUnitPerformanceTest.java

```diff
@@ -0,0 +1,138 @@
+package org.corfudb.performancetest;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.corfudb.protocols.wireprotocol.DataType;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.LogUnitClient;
+import org.corfudb.util.serializer.Serializers;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+
+/**
+ * Created by Nan Zhang and Lin Dong on 10/23/19.
+ */
+
+public class LogUnitPerformanceTest {
+    private String endPoint = "localhost:9000";
+    private int metricsPort = 1000;
+    private long seed = 1024;
+    private int randomBoundary = 100;
+    private Random random;
+    static MetricRegistry metricRegistry;
+    Timer readTimer;
+    Timer writeTimer;
```

Comment:
private

Reply:
You can change LogUnit and Stream directly. I just move them from integration test and try to make them work.

Reply:
I will double check it after you have an abstract class

Reply:
I've add the PerformanceTest super class.

Reply:
Updated, there are some typos in abstract class. Please fix and add comments for tests.